development/kops/run_sonobuoy.sh

```diff
@@ -22,17 +22,17 @@ $PREFLIGHT_CHECK_PASSED || exit 1
 echo "Download sonobuoy"
 if [ "$(uname)" == "Darwin" ]
 then
-  SONOBUOY=https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.19.0/sonobuoy_0.19.0_darwin_amd64.tar.gz
+  SONOBUOY=https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.50.0/sonobuoy_0.50.0_darwin_amd64.tar.gz
```

Comment:
Someday, it would be nice to automate this and use a file to contain the version like GIT_TAG